main.sh

```diff
@@ -355,6 +356,7 @@ register_binfmt() {
 
 setup_linux_host() {
     apt_packages=()
+    IFS=" " read -r -a apt_packages <<<"${packages}"
```

Comment:
```suggestion
    while read -rd,; do
        if [[ -n "${REPLY}" ]]; then
            apt_packages+=("${REPLY}")
        fi
    done <<<"${packages// /,},"
```

(The above should fix IFS issue and allow both space and comma separated list.)